code/game/jobs/job/outsider/representative.dm

```diff
@@ -42,7 +42,7 @@
 	l_hand =  /obj/item/storage/briefcase
 	backpack_contents = list(
 		/obj/item/device/camera = 1,
-		/obj/item/gun/energy/pistol = 1
+		/obj/item/gun/energy/disruptorpistol = 1
```

Comment:
Should remain a regular energy pistol too.

Reply:
```suggestion
		/obj/item/gun/energy/pistol = 1
```